### PR TITLE
Phillip2020epidemiological

### PR DESCRIPTION
**Pull request by haisuzhang**
Initial Try

### CODE REVIEW

File: data/phillip2020epidemiological/phillip2020epidemiological.yaml [hunk 1]
```diff
@@ -0,0 +1,1061 @@
...
+  \ investigation of outpatient and mild coronavirus disease 2019 cases. \n"
+analytes:
+  N1_and_N2_probes:
+    description: >
+      SARS-CoV-2 RNA genome copy concentration calculated from evaluation of both
+      N1 and N2 probe.
+    specimen: nasopharyngeal_swab
+    biomarker: SARS-CoV-2
+    gene_target: N1 and N2 probe
+    limit_of_quantification: '3162'
```
Comment (tillahoffmann) [inline data/phillip2020epidemiological/phillip2020epidemiological.yaml:16]: I think the test currently fail because this is a string rather than a number.

Comment (haisuzhang) [reply data/phillip2020epidemiological/phillip2020epidemiological.yaml]: Fixed with actual number. The LOD/LOQ is described as: "with a limit of detection in the range of 1000–3162 viral RNA copies per mL". While in the challenger paper, the LOD is filled with 67.32 (Calculated with ct=40). Any thought on the LOD/LOQ in yaml file?

**Review by YWAN446 — COMMENTED** [review]
I added some comments and please work on generating the proper format Markdown file with text blocks.

File: data/phillip2020epidemiological/phillip2020epidemiological-extraction.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+```python
+phillip2020 = dict(title="Epidemiological Correlates of Polymerase Chain Reaction Cycle Threshold Values in the Detection of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2)",
+               doi="10.1093/cid/ciaa1469",
+               description=folded_str('This study was conducted in Utah and Wisconsin between 23 March and 13 May 2020, with testing data collected during a prospective household transmission investigation of outpatient and mild coronavirus disease 2019 cases. \n'),
+               analytes=dict(N1_and_N2_probes=dict(description=folded_str("SARS-CoV-2 RNA genome copy concentration calculated from evaluation of both N1 and N2 probe.\n"),
+                                                    specimen="nasopharyngeal_swab",
+                                                    biomarker="SARS-CoV-2",
+                                                    gene_target="N1 and N2 probe",
+                                                    limit_of_quantification=3162,
+                                                    limit_of_detection=3162,
```
[the remark this answers is not in the document]
Comment (haisuzhang) [reply data/phillip2020epidemiological/phillip2020epidemiological-extraction.md:64]: Added

File: data/phillip2020epidemiological/phillip2020epidemiological.yaml [hunk 3]
```diff
@@ -0,0 +1,1061 @@
...
+    description: >
+      SARS-CoV-2 RNA genome copy concentration calculated from evaluation of both
+      N1 and N2 probe.
+    specimen: nasopharyngeal_swab
+    biomarker: SARS-CoV-2
+    gene_target: N1 and N2 probe
+    limit_of_quantification: 3162
+    limit_of_detection: 3162
+    unit: gc/mL
+    reference_event: enrollment
```
[the remark this answers is not in the document]
Comment (haisuzhang) [reply data/phillip2020epidemiological/phillip2020epidemiological.yaml:19]: changed to symptom onset.

File: data/phillip2020epidemiological/phillip2020epidemiological.yaml [hunk 4]
```diff
@@ -0,0 +1,1061 @@
+# yaml-language-server: $schema=../.schema.yaml
+title: Epidemiological Correlates of Polymerase Chain Reaction Cycle Threshold Values
+  in the Detection of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2)
+doi: 10.1093/cid/ciaa1469
+description: "This study was conducted in Utah and Wisconsin between 23 March and\
```
[the remark this answers is not in the document]
Comment (haisuzhang) [reply data/phillip2020epidemiological/phillip2020epidemiological.yaml:5]: Fixed

File: data/phillip2020epidemiological/phillip2020epidemiological-extraction.md
[the remark this answers is not in the document]
Comment (haisuzhang) [reply data/phillip2020epidemiological/phillip2020epidemiological-extraction.md:1]: Changed

**Review by YWAN446 — APPROVED** [review]
Looks good!

**Pull request merged**